src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -694,7 +732,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
         case _ => Nil
       }
       val dependencies = upstream.flatten ++ downstream
-      val dependenciesResolved = dependencies.isEmpty || dependencies.forall(isDone)
+      val dependenciesResolved = dependencies.isEmpty || dependencies.forall(d => isDone(d) || isPreempted(d))
```

Comment:
Yes and no, when a call gets preempted we immediately re-insert a clone of this call with the `NotStarted` Status (and attempt incremented) before calling `tryStartingRunnableCalls` again. Because the copy has the same `Call` object, calls that were depending on the preempted call will be depending on the new clone so they won't start yet.
A comment could be useful though to make this clearer.